### ISSUE TITLE
Music Assistant can not connect with Subsonic API

### ISSUE THREAD

**Issue by sebmuc99** [ampache/ampache#3952]
The Music Assistant Server (https://github.com/music-assistant/server) can not be connected with the Subsonic API

see details here: https://github.com/music-assistant/hass-music-assistant/issues/2086

**Comment by lachlan-00** [comment]
I still haven't had time to put into opensubsonic extensions like multi artist but will be when i can.

I do want to support all the extensions

**Comment by troycarpenter** [comment]
The 7.0.0 release notes says:
"Subonic
Add openSubsonic extensions [ampache.org](https://ampache.org/api/subsonic/#opensubsonic-api-extension)"

Does that mean that Music Assistant should work now?

**Comment by lachlan-00** [comment]
I've started.
 https://ampache.org/api/subsonic/

I've done the basics only on the new endpoints (except getLyricsBySongId))

I'll be moving onto endpoint extras like multi artist. the issue i have at the moment is the data returns aren't always correct so i need to do a bit of a rewrite which i've been saving till now.

**Comment by lachlan-00** [comment]
I'll be documenting it a bit better and will be extending my testing for subsonic.

Right now the class is pretty poor so it's a big job to clean up

**Comment by lachlan-00** [comment]
I've released my full rewrite in opensubsonic and all the current extensions are supported.

there may be missing additional data fields but that's WIP to verify.

I've fully synced my home library with Symfonium which is one of the stricter clients as well

**Comment by lachlan-00** [comment]
Missing Subsonic base parameters means its missing one of these

<img width="522" height="299" alt="Image" src="https://github.com/user-attachments/assets/b134742a-8645-4108-9128-a9da9ba90a92" />

What is the url sent by the ping?

**Comment by lachlan-00** [comment]
you're right, i'm not getting any post variables. it's in the code but i'm not seeing them get to the class.

**Comment by lachlan-00** [comment]
Post data wasn't checking auth variables, that's fixed after 007f79570153f4b98595759a767f12b04b3353e2

I didn't realise there are 2 sections in subsonic classes getting the variables and then 3 areas where auth could come from :/

point release next week but if you want to bypass the simplest wat to fix is to turn off the post feature by deleting the line from the api extensions.

files in src/Module/Api/

<img width="1594" height="678" alt="Image" src="https://github.com/user-attachments/assets/51889836-4f17-48ed-8642-db43af45d89c" />

**Comment by JoeDat** [comment]
Confirmed the issue is resolved in Music Assistant when testing against the develop branch.  Nice work as always @fufroma  and @lachlan-00.   I'm not the OP,  but this issue can probably be closed.

**Comment by lachlan-00** [comment]
full release with the post fix is out now.

**Comment by lachlan-00** [comment]
we are soooooo compliant
